doc/_extensions/zephyr/doxytooltip/static/tippy/popper.min.js

```diff
@@ -0,0 +1,8 @@
+//# sourceMappingURL=popper.min.js.map
```

Comment:
Do we also want the `.map` files to debug potential issues?

Reply:
I lean towards no (mostly because these are two 100KB+ files to add to the repo) but don't really have a strong opinion. @gmarull thoughts?